includes/lib/transifex-live-integration-prerender.php

```diff
@@ -158,8 +158,18 @@ function call_curl( $url ) {
 	function callback( $buffer ) {
 		global $wp;
 		$output = $buffer;
+
+		# Try to get language code from the Transifex-Lang header.
+		# This is used in the instance where subdomains have been configured
+		# outside Wordpress so it does not have any knowledge of other
+		# subdomains.
+		$lang = urlencode($_SERVER['HTTP_X_TRANSIFEX_LANG']);
+
 		$debug_html = '<!--' . "\n";
 		$page_url = home_url( $wp->request );
+		if ( !empty($lang) ) {
+			$page_url = str_replace('www', $lang, $page_url);
+		}
```

Comment:
What if the url doesn't include `www`? Naked domains such as `github.com` will not work it seems. Also, if i'm not mistaken this will replace all occurrences of `www` wherever they may appear in the URL. E.g. `http://www.website.com/the-www-is-great` will become `http://fr.website.com/the-fr-is-great`.

Reply:
Tested that. You are right.

Reply:
@kouk @rigaspapas I've added a bit more clever utility function to do this. Plz take another look.